query-connector/src/app/signin/page.tsx

```diff
@@ -0,0 +1,84 @@
+"use client";
```

Comment:
Why are we overriding the default behavior of Next to render on the server here in favor of rendering on the client side? Unless we have a good reason this should be removed.